pkg/querier/blocks_store_queryable.go

```diff
@@ -3,6 +3,8 @@ package querier
 import (
 	"context"
 	"fmt"
+	"github.com/cortexproject/cortex/pkg/cortexpb"
```

Comment:
We group imports. Please configure GoLand File Watchers as follow:

![Uploading Screenshot 2021-05-13 at 17.53.52.png…]()